lib/display/display.ex

```diff
@@ -4,18 +4,31 @@ defmodule Inky.Display do
   """
 
   alias Inky.LookupTables
-  
+
   @type t() :: %__MODULE__{}
 
-  @enforce_keys [:type, :width, :height, :packed_dimensions, :rotation, :accent, :luts]
```

Comment:
If these are required for one display but not the other, we should consider refactoring things so that we don't force re-usability where it is not required/meaningful (maybe that's exactly what you did here? :D). "False generalisation" only makes things unclear since you don't knew when/if something is required/missing.

Reply:
I've reverted this change too, since these keys can peacefully coexist in the struct the defines the impression display. We're not using packed dimensions because it's redundant/unnecessary, but if someone wants to use them in the future, it'll be possible.

As for luts, it's not used at all for the impression. Not sure we need to define a separate struct+key enforcement just for that though.

Reply:
This is good food for thought, though... if the devices are diverting, perhaps our configurations should, too.

One way of dealing with it would be to have a protocol common for all types, for the things that are common to all displays and then have the specific drivers check if it is a value of the correct underlying type at runtime during init. But that might be a bit more than what you signed up for at this point and a bit of over-engineering before we have several displays where the fit is off... thoughts?

Reply:
That sounds like it might be a worthwhile refactor to open a separate issue for.